README.md

```diff
@@ -1 +1,37 @@
-# ReviewRepo
\ No newline at end of file
+## Welcome to GitHub Pages
+
+You can use the [editor on GitHub](https://github.com/r-ahmed2022/Youtube-Clone/edit/master/README.md) to maintain and preview the content for your website in Markdown files.
+
+Whenever you commit to this repository, GitHub Pages will run [Jekyll](https://jekyllrb.com/) to rebuild the pages in your site, from the content in your Markdown files.
+
+### Markdown
+
+Markdown is a lightweight and easy-to-use syntax for styling your writing. It includes conventions for
+
+```markdown
+Syntax highlighted code block
```

Comment:
Readme file should be descriptive enough to give a general information about your project

Reply:
- [ ] Kindly make sure to provide kind and suggestive reviews instead of giving direct orders.
> Example: I think it would be better if you could format your code properly to make them readable.